group-coordinator/src/test/java/org/apache/kafka/coordinator/group/GroupMetadataManagerTest.java

```diff
@@ -231,14 +263,20 @@ public List<Record> build(TopicsImage topicsImage) {
     static class GroupMetadataManagerTestContext {
         static class Builder {
             final private MockTime time = new MockTime();
-            final private MockCoordinatorTimer<Record> timer = new MockCoordinatorTimer<>(time);
+            final private MockCoordinatorTimer<Void, Record> timer = new MockCoordinatorTimer<>(time);
             final private LogContext logContext = new LogContext();
             final private SnapshotRegistry snapshotRegistry = new SnapshotRegistry(logContext);
             private MetadataImage metadataImage;
-            private List<PartitionAssignor> assignors;
+            private List<PartitionAssignor> assignors = Collections.singletonList(new MockPartitionAssignor("range"));
             private List<ConsumerGroupBuilder> consumerGroupBuilders = new ArrayList<>();
             private int consumerGroupMaxSize = Integer.MAX_VALUE;
             private int consumerGroupMetadataRefreshIntervalMs = Integer.MAX_VALUE;
+            private final TopicPartition groupMetadataTopicPartition = new TopicPartition("topic", 0);
```

Comment:
nit: Could we move this next to the other final private attributes? Could we also invest private final to be consistent with the others?